### PR TITLE
Add adherent mandates (for committees, territorial councils)

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

**Comment by illusionOfParadise** [comment]
[ch12054]

**Comment by illusionOfParadise** [comment]
[ch12052]

File: src/Repository/AdherentMandate/CommitteeAdherentMandateRepository.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            ->leftJoin('mandate.adherent', 'adherent')
+            ->where('mandate.committee = :committee')
+            ->andWhere('mandate.finishAt IS NULL')
+            ->setParameter('committee', $committee)
+            ->getQuery()
+            ->getResult()
+        ;
+
+        return \array_values(\array_map(static function (CommitteeAdherentMandate $mandate) {
+            return $mandate->getAdherent()->getId();
```
Comment (ottaviano) [inline src/Repository/AdherentMandate/CommitteeAdherentMandateRepository.php:66]: `select(adherent.id)` + `getScalarResult()` ?

**Review by ottaviano — APPROVED** [review]
👍

**Pull request merged**